src/main/java/com/slickqa/webdriver/InFlow.java

```diff
@@ -2,37 +2,37 @@
 
 /**
  * The InFlow interface enforces 3 methods:
- * 1- if the browser's current page is itself
- * 2- how to handle any forms on the page
- * 3- how to continue past the page.
- * To help provide information to accomplish this when you implement this interface you will need to define the type of context object that is used to handle and complete the page.
+ * 1- if the current step is itself
+ * 2- how to handle any actions on the step
+ * 3- how to continue past the step.
+ * To help provide information to accomplish this when you implement this interface you will need to define the type of context object that is used to handle and complete the step.
  *
  * @author slambson
  */
 public interface InFlow<T> {
 
     /**
-     * Determine if the browser's current page is the one represented by this class.  It is VERY important that this
+     * Determine if the current step is the one represented by this class.  It is VERY important that this
      * be a quick check, not one that waits for 30 seconds before returning false.  You should pass in a 0 timeout to
      * whatever elements you look for in the browser.
```

Comment:
Still referencing browser... maybe should update that as well?

Reply:
Missed that... :)

Reply:
It is fixed and checked in.